diff --git a/GammaGear/MainWindow.xaml b/GammaGear/MainWindow.xaml
index 3b6473e..ae3b9b4 100644
--- a/GammaGear/MainWindow.xaml
+++ b/GammaGear/MainWindow.xaml
@@ -618,7 +618,6 @@
                                 <StackPanel.Resources>
                                     <Style TargetType="TextBlock">
                                         <Setter Property="Margin" Value="3 3 3 0"/>
-                                        <Setter Property="TextAlignment" Value="Center"/>
                                     </Style>
                                 </StackPanel.Resources>
                                 <TextBlock FontWeight="Bold">
@@ -640,7 +639,6 @@
                                     <StackPanel.Resources>
                                         <Style TargetType="TextBlock">
                                             <Setter Property="Margin" Value="3 3 3 0"/>
-                                            <Setter Property="TextAlignment" Value="Center"/>
                                         </Style>
                                     </StackPanel.Resources>
                                     <TextBlock FontWeight="Bold">
@@ -705,7 +703,7 @@
             <StatusBar DockPanel.Dock="Bottom" Height="20" VerticalAlignment="Bottom">
                 <StatusBarItem>
                     <TextBlock>
-                        Items Loaded: 42069
+                        Items Loaded: 0
                     </TextBlock>
                 </StatusBarItem>
             </StatusBar>
diff --git a/GammaGear/MainWindow.xaml.cs b/GammaGear/MainWindow.xaml.cs
index e674505..2808740 100644
--- a/GammaGear/MainWindow.xaml.cs
+++ b/GammaGear/MainWindow.xaml.cs
@@ -232,7 +232,7 @@ namespace GammaGear
         private void SetItemContent(ItemDisplay item, StackPanel parent)
         {
             parent.Children.Clear();
-            TextBlock tb = item.GetStatDisplay(showDBItemIDs);
+            TextBlock tb = item.GetStatDisplay(false, showDBItemIDs);
             parent.Children.Add(tb);
         }
         private void DBItemSelected(object sender, RoutedEventArgs eventArgs)
@@ -351,6 +351,10 @@ namespace GammaGear
             WizardPetBox.Content    = mainLoadout.GetEquippedFromType(Item.ItemType.Pet)?.Name ?? "None";
             WizardDeckBox.Content   = mainLoadout.GetEquippedFromType(Item.ItemType.Deck)?.Name ?? "None";
             WizardMountBox.Content  = mainLoadout.GetEquippedFromType(Item.ItemType.Mount)?.Name ?? "None";
+
+            // Update the button's tooltip
+            WizardHatBox.ToolTip = mainLoadout.GetEquippedFromType(Item.ItemType.Hat)?.GetStatDisplay(true, showDBItemIDs) ?? null;
+
         }
         private void TabControlSelectionChanged(object sender, SelectionChangedEventArgs eventArgs)
         {
@@ -659,13 +663,14 @@ namespace GammaGear
             { "NoTrade",      new BitmapImage(new Uri(@"pack://application:,,,/GammaGear;component/Assets/Images/(Icon)_Flag_NoTrade.png", UriKind.Absolute)) },
         };
 
-        public TextBlock GetStatDisplay(bool showIDs)
+        public TextBlock GetStatDisplay(bool showName, bool showIDs)
         {
-            void AddSingle(TextBlock tb, string before, BitmapImage img1 = null, BitmapImage img2 = null, string after = null)
+            void AddSingle(TextBlock tb, string before, BitmapImage img1 = null, BitmapImage img2 = null, string after = null, bool boldedText = false)
             {
                 if (!string.IsNullOrEmpty(before))
                 {
                     Run run = new Run(before);
+                    if (boldedText) run.FontWeight = FontWeights.Bold;
                     tb.Inlines.Add(run);
                 }
                 if (img1 != null)
@@ -709,7 +714,15 @@ namespace GammaGear
                 }
             }
 
-            TextBlock tb = new TextBlock();
+            TextBlock tb = new TextBlock()
+            {
+                Margin = new Thickness(3, 3, 3, 0)
+            };
+
+            if (showName)
+            {
+                AddSingle(tb, Name, null, null, "\n", true);
+            }
 
             if (showIDs)
             {
